.npmrc

```diff
@@ -0,0 +1,2 @@
+@sbac:registry=https://airdev.jfrog.io/airdev/api/npm/npm
+//airdev.jfrog.io/airdev/api/npm/:_authToken=${NPM_AUTH_TOKEN}
```

Comment:
NPM_AUTH_TOKEN is an environment variable and the CI server sets it on the build so it can publish.

Reply:
I put the auth token on Confluence in case it is needed elsewhere.